src/outfit/free.ts

```diff
@@ -0,0 +1,60 @@
+import { Outfit, OutfitSpec } from "grimoire-kolmafia";
+import { Location, toJson } from "kolmafia";
+import { $familiar, $familiars, $item, $items, get } from "libram";
+import { freeFightFamiliar } from "../familiar";
+import { chooseBjorn } from "./bjorn";
+import { bonusGear } from "./dropsgear";
+import { BonusEquipMode, cleaverCheck, validateGarbageFoldable } from "./lib";
+
+type MenuOptions = {
+  canChooseMacro?: boolean;
+  location?: Location;
+  includeExperienceFamiliars?: boolean;
+  allowAttackFamiliars?: boolean;
+};
+export function freeFightOutfit(spec: OutfitSpec = {}, options: MenuOptions = {}): Outfit {
+  cleaverCheck();
+  validateGarbageFoldable(spec);
+  const outfit = Outfit.from(
+    spec,
+    new Error(`Failed to construct outfit from spec ${toJson(spec)}!`)
+  );
+
+  outfit.familiar ??= freeFightFamiliar(options);
+  const mode =
+    outfit.familiar === $familiar`Machine Elf` ? BonusEquipMode.DMT : BonusEquipMode.FREE;
+  outfit.modifier.push(
+    $familiars`Pocket Professor, Grey Goose`.includes(outfit.familiar)
+      ? "Familiar Experience"
+      : "Familiar Weight"
+  );
+
+  const bjornChoice = chooseBjorn(mode, outfit.familiar);
+
+  if (get("_vampyreCloakeFormUses") < 10) outfit.equip($item`vampyric cloake`);
+  outfit.bonuses = bonusGear(mode);
+
+  if (outfit.familiar !== $familiar`Grey Goose`) outfit.setBonus($item`tiny stillsuit`, 500);
```

Comment:
Shouldn't the pocket professor also be excluded? I think the familiar experience is more useful for it.